CONST_versions.txt

```diff
@@ -8,44 +8,44 @@ future==0.16.0
 generateDS==2.29.24
 GeoAlchemy2==0.5.0
 geojson==2.4.0
-geolink-formatter==1.3.0
+geolink-formatter==1.3.1
 httplib2==0.11.3
 hupper==1.3
 idna==2.6
 Jinja2==2.10
 jsonschema==2.6.0
 JSTools==1.0
-lxml==4.2.4
+lxml==4.3.0
 Mako==1.0.7
 MarkupSafe==1.0
 netifaces==0.10.7
 papyrus==2.3
 PasteDeploy==1.5.2
 plaster==1.0
 plaster-pastedeploy==0.5
-psycopg2==2.7.5
+psycopg2==2.7.7
 pyconizer==0.1.4
 Pygments==2.2.0
 pyproj==1.9.5.1
-pyramid==1.9.2
-pyramid-debugtoolbar==4.4
+pyramid==1.10.2
+pyramid-debugtoolbar==4.5
 pyramid-mako==1.0.2
 pyramid-tm==2.2
 pyreproj==1.0.0
-PyYAML==3.13
+PyYAML==4.2b4
```

Comment:
Are you sure to want to use a beta release ?

Reply:
yes, as it's the needed version for pyramid_oereb v1.3.1, but anyhow, this is a mere 'snapshot state' PR as the next version PR for pyramid_oereb 1.4.x is already in the pipeline and coming today :)